### PR TITLE
feat: added code blocks with highlights

### PR DESCRIPTION
**Pull request by Kenny-Gin1**
- Added code blocks

### CODE REVIEW

**Review by ericzakariasson — APPROVED** [review]
Looks really nice, good job!

File: apps/web/src/components/Editor.tsx [hunk 1]
```diff
@@ -13,6 +14,11 @@ import { Command } from "../lib/tiptap/command/command-extension";
 import { commandSuggestions } from "../lib/tiptap/command/command-suggestions";
 import { getCommandExtensions } from "../lib/tiptap/tiptap";
 import { Toggle } from "./ui/toggle";
+import css from "highlight.js/lib/languages/css";
+import js from "highlight.js/lib/languages/javascript";
+import ts from "highlight.js/lib/languages/typescript";
+import html from "highlight.js/lib/languages/xml";
```
Comment (philipandersson) [inline apps/web/src/components/Editor.tsx:20]: Does `highlight.js` support any other languages?
Would be nice to be able to embed Rust, Haskell, C etc. and especially Solidity given our crypto niche.

**Review by philipandersson — APPROVED** [review]
LGTM

File: apps/web/src/components/Editor.tsx (hunk 1 above)
Comment (Kenny-Gin1) [reply apps/web/src/components/Editor.tsx:20]: Yeah, a bunch (including the ones you mentioned) except Solidity it seems like. And most of them should already be available without the `highlight.register` functionality: https://www.npmjs.com/package/lowlight#types

**Pull request merged**